src/EVM/SymExec.hs

```diff
@@ -283,7 +283,8 @@ interpret fetcher maxIter askSmtIters heuristic vm =
         evalRight <- toIO $ do
           (rb, vmb) <- liftIO $ stToIO $ runStateT (continue False) vm { result = Nothing }
           interpret fetcher maxIter askSmtIters heuristic vmb (k rb)
-        (a, b) <- liftIO $ concurrently evalLeft evalRight
+        a <- liftIO evalLeft
+        b <- liftIO evalRight
```

Comment:
Just marking this as THE important change.
Apparently, the parallel run was never working properly?

Reply:
Yes, that's my suspicion. Because the ` -threaded -with-rstopts=-N` was never passed, so this never ran concurrently, until I passed those via `-f devel`, and pooof it then triggered. There is some kind of race condition there for sure. With this change, it never fails.